docs/raspberry_pi_os_build_instructions.md

```diff
@@ -1,4 +1,6 @@
-# Manual Installation Instructions
+# Raspberry Pi OS Build Instructions
```

Comment:
Maybe clarify in the title, something like:
"Local Dev Build Instructions"

(the specific OS being used isn't really relevant for the title. That's just a detail to be explained / a step to be followed below)